docs/Makefile

```diff
@@ -53,3 +53,25 @@ html: cpp-apidoc spark-apidoc
 				--quiet
 	@echo
 	@echo "Build finished. The HTML pages are in $(BUILDDIR)/html."
+
+.PHONY: pyspark-apidoc
+pyspark-apidoc:
+	cd $(ROOTDIR)/pyspark && \
+	poetry run sphinx-apidoc -o $(ROOTDIR)/docs/pyspark/api graphar_pyspark/
```

Comment:
It's OK to assume that `sphinx-apidoc` already exist, no need to introduce `poetry`

Reply:
I answered in the next comment. Generally speaking it is about the same thing.